src/mo_types/type.ml

```diff
@@ -916,14 +916,36 @@ exception Undecided
 
 module SS = Set.Make (OrdPair)
 
-let max_depth = 10_000
+module RelArg :
+  sig
+    type arg
+    val default : arg (* ordinary subtyping, with loss of info *)
+    val loss_less : arg (* loss_less subtyping *)
```

Comment:
I'd drop the underscore (line 925 too). "Lossless" is a word now. Like "fearless".